packages/adapter-netlify/index.js

```diff
@@ -1,7 +1,6 @@
 import { appendFileSync, existsSync, readFileSync, writeFileSync } from 'fs';
 import { dirname, join, resolve, posix } from 'path';
 import { fileURLToPath } from 'url';
-import glob from 'tiny-glob/sync.js';
```

Comment:
we can remove `tiny-glob` from `packages/adapter-netlify/package.json` and then run `pnpm install` to update the `pnpm-lock.yaml` file since `tiny-glob` is no longer used

Reply:
Done in https://github.com/sveltejs/kit/pull/6666/commits/8ebd7f93c5d2a30afe236ae70196ad3d951793e0.